### PR TITLE
Fix a typo and a tooltip message in Diagnostics

### PR DESCRIPTION
**Pull request by qrazi**

### CODE REVIEW

File: src/helpers/DiagnosticsHelper.php
Comment (bencroker) [inline src/helpers/DiagnosticsHelper.php:1]: Can you explain why you removed `refreshCacheAutomaticallyForGlobals` and replaced with `refreshCacheWhenElementSavedUnchanged`? This is completely different type of recommendation.

Comment (qrazi) [reply src/helpers/DiagnosticsHelper.php:1]: So, at line 364, this is not a removal of `refreshCacheAutomaticallyForGlobals`, but making the `'info'` message fit with the setting that is checked - which is `refreshCacheWhenElementSavedUnchanged`. I suspect this was accidentally wrongly copy pasted during the refactoring in 9fd7001. 

Or do I misunderstand your question? :sweat_smile:

Comment (bencroker) [reply src/helpers/DiagnosticsHelper.php:1]: Oh I see now, this was a mistake on my part, I’ll fix.

**Comment by bencroker** [comment]
Thanks for the PR!

**Comment by bencroker** [comment]
Released in [4.14.0](https://github.com/putyourlightson/craft-blitz/releases/tag/4.14.0).

**Pull request merged**